tests/unit/frameworks/continuous_learning/multihead/multihead_test.py

```diff
@@ -0,0 +1,95 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2020, Numenta, Inc.  Unless you have an agreement
+# with Numenta, Inc., for a separate license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+"""
+Unit tests to ensure that target variables are being transformed properly according to
+the continual learning setup found in the benchmarks repository:
+https://github.com/GMvandeVen/continual-learning
+"""
+
+import unittest
+
+from torch.utils.data import DataLoader
+
+from nupic.research.frameworks.continuous_learning.multihead.subdataset import (
+    get_datasets,
+)
+
+
+def setup(dataset_name, scenario):
+    train_datasets, test_datasets = get_datasets(dataset_name, scenario=scenario)
```

Comment:
This test requires downloading MNIST dataset to run which could take a long time. Unit tests are supposed to be fast and only check for specific functionality.
If you need a dataset to test you should use [FakeDataLoader](https://github.com/numenta/nupic.research/blob/53c19f5eec2da8091219f270f124c5aef74bbe02/nupic/research/frameworks/pytorch/test_utils/fake_data.py#L30) instead

Reply:
Since the target transforms interface with the data class directly, I think I'll need to deal with [FakeData](https://pytorch.org/docs/stable/torchvision/datasets.html#fakedata) instead of the data loader.